database/migrations/2025_02_22_174047_create_resource_media_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('resources_media', function (Blueprint $table) {
+            $table->id();
+            $table->morphs('model');
+            $table->string('path');
```

Comment:
Recorda agregar la longitud, 50 seria suficiente para el path
```suggestion
            $table->string('path', 50);
```